### ISSUE TITLE
Testnet storage state mismatch at block 460271

### ISSUE THREAD

**Issue by roman-khimov** [nspcc-dev/neo-go#965]
Very early:
file BlockStorage_500000/dump-block-461000.json: block 460271, changes number mismatch: 1 vs 0

There is just one invocation there. Looks like we're running something successfully when C# node fails. Application log from C# node:
```
{
   "jsonrpc" : "2.0",
   "id" : 1,
   "result" : {
      "txid" : "0x836487ae75e331fa53ccd4bc2a1674fbbd4d732143d2fe43648ffa753e66a1ea",
      "executions" : [
         {
            "vmstate" : "FAULT",
            "trigger" : "Application",
            "gas_consumed" : "0.059",
            "contract" : "0x7f0fc66aa2197c3f6510d2550fd41d35eeef188f",
            "notifications" : [],
            "stack" : []
         }
      ]
   }
}
```
neo-go:
```
{
   "result" : {
      "txid" : "0x836487ae75e331fa53ccd4bc2a1674fbbd4d732143d2fe43648ffa753e66a1ea",
      "executions" : [
         {
            "stack" : [
               {
                  "value" : "1",
                  "type" : "Integer"
               }
            ],
            "contract" : "0x7f0fc66aa2197c3f6510d2550fd41d35eeef188f",
            "notifications" : [],
            "gas_consumed" : "1.108",
            "trigger" : "Application",
            "vmstate" : "HALT"
         }
      ]
   },
   "jsonrpc" : "2.0",
   "id" : 1
}
```

**Comment by roman-khimov** [comment]
Quite simple contract:
```
{
   "jsonrpc" : "2.0",
   "id" : 1,
   "result" : {
      "version" : 0,
      "name" : "Test15",
      "email" : "joioo",
      "code_version" : "1",
      "description" : "jojoo",
      "script" : "56c56b6c766b00527ac46c766b51527ac4616c766b00c30d696e666c6174696f6e52617465876c766b52527ac46c766b52c3644900616c766b51c3c0519c009c6c766b54527ac46c766b54c3640e00006c766b55527ac46230006c766b51c300c36c766b53527ac46c766b53c3616522006c766b55527ac4620f0061006c766b55527ac46203006c766b55c3616c756652c56b6c766b00527ac4616168164e656f2e53746f726167652e476574436f6e746578740d696e666c6174696f6e526174656c766b00c3615272680f4e656f2e53746f726167652e50757461516c766b51527ac46203006c766b51c3616c7566",
      "returntype" : "ByteArray",
      "properties" : {
         "storage" : true,
         "is_payable" : false,
         "dynamic_invoke" : false
      },
      "author" : "Peter",
      "parameters" : [
         "ByteArray",
         "Array"
      ],
      "hash" : "0xe3c013fa2210e50af2b8010dcd1854ae2362b126"
   }
}
```
Called with ("inflationRate", 2) parameters.

**Comment by roman-khimov** [comment]
PICKITEM with [0, 2] stack. Wow.
```
NEO-GO-VM 94 > estack
[
    {
        "type": "Integer",
        "value": "2"
    }
]
NEO-GO-VM 94 > step
at 95 (PUSH0)
NEO-GO-VM 95 > step
at 96 (PICKITEM)
NEO-GO-VM 96 > estack
[
    {
        "type": "Integer",
        "value": "2"
    }
]
```

**Comment by roman-khimov** [comment]
bafdb916a0bd9a5408e7d98b95c1a038305a8773 is the culprit.

**Comment by roman-khimov** [comment]
Fixed by #967.
